ush/global_cycle_driver.sh

```diff
@@ -95,15 +95,15 @@ for n in $(seq 1 $ntiles); do
     ln -fs $FIXorog/${CASE}/C${CRES}.mx${OCNRES}_oro_data.tile${n}.nc   $DATA/fnorog.00$n
   fi
 
-  if [[ "$DO_SNO_INC_JEDI" == ".true." ]] ; then  
+  if [[ "$DO_SNO_INC" == ".true." ]] ; then  
         ln -fs $COMIN/$PDY.${cyc}0000.xainc.tile${n}.nc      $DATA/snow_xainc.00$n
   fi
 
-  if [[ "$DO_SOI_INC_JEDI" == ".true." ]] ; then
+  if [ "$DO_SOI_INC" == ".true." ] && [ "$GCYCLE_INTERP_LNDINC" == ".false." ] ; then
         ln -fs $COMIN/soil_sfcincr_jedi.00$n $DATA/soil_xainc.00$n
   fi
 
-  if [[ "$DO_SOI_INC_GSI" == ".true." ]] ; then
+  if [ "$DO_SOI_INC" == ".true." ] && [ "$GCYCLE_INTERP_LNDINC" == ".true." ] ; then
```

Comment:
When I run the new regression test, it points to the `soil_sfcincr_jedi` files. There is no logic to pick the tiled gsi data.